azure-functions/acbs-function/mappings/facility/facility-fixed-fee-amend.js

```diff
@@ -0,0 +1,42 @@
+const CONSTANTS = require('../../constants');
+const { now } = require('../../helpers/date');
+
+const { FACILITY } = CONSTANTS;
+
+const facilityFixedFeeAmend = (amendments) => {
+  try {
+    const records = [];
+    const { amendment } = amendments;
+    const { amount } = amendment;
+
+    // 1. UKEF Exposure
+    if (amount) {
```

Comment:
I know this code is generally non-changable in terms of style due to the importance of keeping it all in sync, so I'll avoid commenting things for readability (for instance here, a guard clause would remove a level of complexity)